db/migrate/20161119134300_add_organizer_to_workshops.rb

```diff
@@ -0,0 +1,5 @@
+class AddOrganizerToWorkshops < ActiveRecord::Migration
+  def change
+    add_column :workshops, :organizer, :string
```

Comment:
I think it is really out of scope for the corresponding user story to unify migrations, which breaks compatibility with earlier versions and would require a database drop.